src/components/conjunction/ConjunctionCollisionProbabilityChart.tsx

```diff
@@ -1,28 +1,30 @@
 import Link from 'next/link';
-import { useTranslations } from 'next-intl';
+import { getTranslations } from 'next-intl/server';
 
+import type { TypeEventSummaryOut } from '@/__generated__/data-contracts';
 import Details from '@/ui/details/details';
 
-import RichText from '../RichText';
+import PocChart from '../charts/poc-chart/PocChart';
 
 type ConjunctionCollisionProbabilityChartProps = {
-  id: string;
+  shortId: string;
+  events: TypeEventSummaryOut[];
 };
 
-const ConjunctionCollisionProbabilityChart = ({ id }: ConjunctionCollisionProbabilityChartProps) => {
-  const t = useTranslations('Accordions.Conjunction.poc_chart');
+const ConjunctionCollisionProbabilityChart = async ({ shortId, events }: ConjunctionCollisionProbabilityChartProps) => {
+  const t = await getTranslations('Accordions.Conjunction.poc_chart');
```

Comment:
why keys from accordion? it should be part of page translations.

Reply:
Fixed